<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gboxFont">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblFontSize">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>Font Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderFontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCustomFont">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Custom Font (Select External Fonts)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFontPath">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUIFont">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Override UI fonts with custom fonts</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkReaderFont">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Override reader fonts with custom fonts</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gboxAdditional">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkClose">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Normal Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAutoTime">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:40px;
    height:40px;
}

QCheckBox::indicator:unchecked{
    image:url(:/res/chk0.png);
}

QCheckBox::indicator:checked{
    image:url(:/res/chk1.png);
}
</string>
        </property>
        <property name="text">
         <string>Auto uses time as the Y-axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDebug">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Debug Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTip">
     <property name="text">
      <string>The above settings will take effect after the app is reopened.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnReStart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Restart the app to make the settings take effect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnBack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
